array_methods.js

```diff
@@ -0,0 +1,186 @@
+
```

Comment:
Overall, Great job on this lab! Strive to complete the bonus problems in your spare time! Looks like you are understanding the various array methods and when best to use them! Keep it up.